src/bin_section.rs

```diff
@@ -1,7 +1,16 @@
 use crate::packed_location::RotatedBy;
 use crate::{BoxSizeHeuristicFn, PackedLocation, RectToInsert, WidthHeightDepth};
-use std::cmp::Ordering;
-use std::fmt::{Display, Formatter};
+
+#[cfg(not(std))]
+use core::{
+    cmp::Ordering,
+    fmt::{Debug, Display, Error as ErrorFmt, Formatter},
+};
+#[cfg(std)]
```

Comment:
Oops I missed something.

`std` just replies on and re-exports `core`. So these `cfg`'s are unnecessary.

This can just be 

```
use core::{
    cmp::Ordering,
    fmt::{Debug, Display, Error as ErrorFmt, Formatter},
};
```

Regardless of whether the `std` feature flag is enabled or not.

`std` only needs to control whether we use `HashMap` or `BTreeMap` as well as whether or not we `impl std::err::Error`

---

This applies to all of the files, and also applies to the `alloc` crate (std just re-exports alloc).

Reply:
cool!